client/ReviewDetailsBox.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Overall, I found that your implementation is great! Your code is very modular which will make it easier for you to navigate through everything. Also, great use of ES6!